Buildings/package.mo

```diff
@@ -314,30 +323,39 @@ have been <b style=\"color:blue\">improved</b> in a
                        <a href=\"https://github.com/lbl-srg/modelica-buildings/issues/3526\">issue 3526</a>.
     </td>
 </tr>
+<tr><td colspan=\"2\"><b>Buildings.Experimental.DHC</b>
+    </td>
+</tr>
+<tr><td valign=\"top\">Buildings.Experimental.DHC.Plants.Combined.Subsystems.BaseClasses.MultipleCommands
+    </td>
+    <td valign=\"top\">The class has been moved to Buildings.Templates.Components.Controls.MultipleCommands.<br/>
```

Comment:
Should we add the change to the conversion script `Resources/Scripts/Conversion/ConvertBuildings_from_10_to_11.0.0.mos`?

Reply:
Yes, please add the change, and then resolve the comment when done.

Reply:
This has already been done in https://github.com/lbl-srg/modelica-buildings/pull/3537/commits/ebf1f9a680b85765b84b7866ea5f7422d8b76050.